coredocument/coredocument.proto

```diff
@@ -198,6 +199,13 @@ message Signed {
   bytes public_key = 5;
 }
 
+// Monetary holds decimal value, id and chain context
+message Monetary {
+  bytes value = 1; // fixed 32 bytes
+  bytes chain = 2; // fixed 4 bytes
+  bytes id = 3; // fixed 32 bytes
```

Comment:
shall we add `proofs.field_length` to this?
Last time I checked, precise proofs didn't support this. Hopefully, that is in already

Reply:
It does support it, although I opted to not use it to not have another dependency on the protobuf syntax.